source/types.ts

```diff
@@ -175,6 +176,15 @@ export type Store = {
 
 export type DraftHeadersVersion = 'draft-6' | 'draft-7'
 
+/**
+ * Validate configuration object for enabling or disabeling specific validations
+ *
+ * The keys must also be keys in the validations object (except `enabled` and `disable`), or `default`
+ */
+export type ValidationsEnabled = {
+	[key in keyof Omit<Validations, 'enabled' | 'disable'> | 'default']?: boolean
```

Comment:
I suppose technically `disable` could be valid here - setting it to `false` would cause validations to run on every request. I don't think I'm ready to support that just yet, though...

Reply:
No I think we shouldn't allow `disable` here, it's better for the users to just specify `false` instead.

Reply:
Also, `default` is not allowed as a key here, but is used in [the readme example](https://github.com/express-rate-limit/express-rate-limit/pull/392/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R528)?

Reply:
> No I think we shouldn't allow disable here, it's better for the users to just specify false instead.

Just to clarify
* `false` = 0 runs
* `true` = 1 run
* `{disable: false}` = Infinity runs 

> Also, default is not allowed as a key here, but is used in [the readme example](https://github.com/express-rate-limit/express-rate-limit/pull/392/files#diff-5a831ea67cf5cf8703b0de46901ab25bd191f56b320053be9332d9a3b0d01d15R528)?

I think `default` is allowed by the `| 'default'` at the end (?)

Reply:
> I think default is allowed by the | 'default' at the end (?)

Oh I misread that line, so sorry!

Reply:
> Just to clarify

> false = 0 runs
> true = 1 run
> {disable: false} = Infinity runs

Oh. It seems weird at first but then kinda makes sense.

I agree with you on doing this sometime later.